src/cityreader/cityreader.py

```diff
@@ -14,15 +27,21 @@
 #
 # Note that the first line of the CSV is header that describes the fields--this
 # should not be loaded into a City object.
+
 cities = []
 
+
 def cityreader(cities=[]):
-  # TODO Implement the functionality to read from the 'cities.csv' file
-  # For each city record, create a new City instance and add it to the 
-  # `cities` list
-    
+    with open('C:\\Users\\Robert Gant\\Desktop\\Folder folder lol\\Python Challenges\\Sprint-Challenge--Intro-Python\\src\\cityreader\\cities.csv', newline='') as f:
```

Comment:
This is a hard coded exact filepath. You want the relative path here, but I'm sure it works all the same.  but `./cities.csv` should have worked all right.